lms/djangoapps/grades/new/course_grade.py

```diff
@@ -240,6 +240,25 @@ def load_persisted_grade(cls, user, course, course_structure):
 
         return course_grade
 
+    @classmethod
+    def get_persisted_grade(cls, user, course):
+        """
+        Gets the persisted grade in the database, without checking
+        whether it is up-to-date with the course's grading policy.
+        For read use only.
+        """
+        try:
+            persistent_grade = PersistentCourseGrade.read_course_grade(user.id, course.id)
+        except PersistentCourseGrade.DoesNotExist:
+            return None
+        course_grade = CourseGrade(user, course, None)  # no course structure needed
```

Comment:
Nitty: Consider putting the rest of this method under an `else:` clause to make the code structure easier to read.
```
try:
except:
else:
```